src/components/Breadcrumb/Breadcrumb.less

```diff
@@ -0,0 +1,43 @@
+.lucid-Breadcrumb {
+
+	@breadcrumb-margin: 7px;
+
+	&-List {
+		list-style-type: none;
+		display: flex;
+		flex-direction: row;
+		flex-wrap: wrap;
+		align-items: center;
+		padding: 0;
+		margin: 0;
+		height: 12px;
+	}
+
+	&-Item {
+		display: flex;
+		flex-direction: row;
+		flex-wrap: wrap;
+		align-items: center;
+		font-size: @size-font;
+		color: @color-textColor;
+
+		a {
+			text-decoration: none;
+			color: @color-primary;
```

Comment:
should be using `@color-linkColor` and `@color-linkColorHover` on hover unless u intentionally left out a hover state